src/client/components/filterToggler/filterToggler.css

```diff
@@ -1,84 +1,78 @@
-.SwitchContainer {
-  font-family: var(--font-ordinary);
+.modesElement {
+  text-align: center;
+  color: var(--mainText-ordinary-white);
+}
 
-  display: -webkit-box;
-  display: -ms-flexbox;
-  display: flex;
-  letter-spacing: 1px;
-  -webkit-box-pack: justify;
-  -ms-flex-pack: justify;
-  justify-content: space-between;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
-  align-items: center;
+.labels {
+  margin: 0 5px;
 }
 
-.Switch {
+.switch {
   position: relative;
   display: inline-block;
   width: 46px;
-  height: 25px;
-  -ms-flex-item-align: center;
-  -ms-grid-row-align: center;
+  height: 20px;
   align-self: center;
 }
 
-.Switch input {
+.switch input {
   display: none;
 }
 
-.Switch input:checked+.Slider {
-  background-color: red;
+.switch input:checked+.slider {
+  background-color: #eee;
 }
 
-.Switch input:checked+.Slider:before {
+.switch input:checked+.slider:before {
   -webkit-transform: translateX(26px);
+  -moz-transform: translateX(26px);
   -ms-transform: translateX(26px);
+  -o-transform: translateX(26px);
   transform: translateX(26px);
-  background: #6080ce;
+  background: #2ECC71;
```

Comment:
I saw you are using variables for colors, can be this color, and also all colors you are using, moved to variables, or use existing one

Reply:
done it